sktime/utils/__init__.py

```diff
@@ -176,3 +178,21 @@ def _get_err_msg(estimator_type):
         return all_estimators
     else:
         return [estimator for (name, estimator) in all_estimators]
+
+
+def _has_tag(Estimator, tag):
+    """Check whether an Estimator has the given tag or not.
+
+    Parameters
+    ----------
+    Estimator : Estimator class
+    tag : str
+        An Estimator tag like "skip-inverse-transform"
+
+    Returns
+    -------
+    bool
+    """
+    assert tag in VALID_ESTIMATOR_TAGS
```

Comment:
I think we should remove the assertion here and write a unit test for it instead. So a unit test that checks if for all estimators, all tags are valid tags. Would that work?

Reply:
I'm also not sure why I used an Estimator class here rather than an instance - that should cause any issue though (I hope).

Reply:
Perhaps even better if we move `VALID_ESTIMATOR_TAGS` somewhere else, into `sktime/base/__init__.py`? That way, we could still have a check on `tag` here.